tests/unittests/graphOptzTest.cpp

```diff
@@ -289,11 +289,25 @@ TEST_F(GraphOptz, cancelTwoTransposes) {
   EXPECT_EQ(F_->getNodes().size(), 2);
 }
 
+TEST_F(GraphOptz, sinkIdentityTranspose) {
```

Comment:
nit: there is no actual sink happening here it's just elimination of identity transpose.

Reply:
Renamed `remove`. For me `sink` was equivalent to `remove`. @rdzhabarov what is the actual meaning of  `sink` in the context of Glow ?

Reply:
@tlepley-cadence sorry for delay, I'm OOO today.
sink means moving a node below the other node, check `Sink Transpose below batch RELU nodes.` in the graph optimizer for example.